packages/common/src/audius-query/types.ts

```diff
@@ -147,13 +156,50 @@ export type QueryHookOptions = {
   debounce?: number
 }
 
+export type PaginatedQueryHookOptions = QueryHookOptions & {
+  /**
+   * Page size to use.
+   * NOTE: if you change this value after the hook was initialized you will cause a cache miss and it will re-fetch data.
+   */
+  pageSize: number
+  /**
+   * Toggles single page data mode. If true the hook will only return the current page's data. Equivalent to usePaginatedQuery in the past
+   */
+  singlePageData?: boolean //
+}
+
 export type QueryHookResults<Data> = {
   data: Data
   status: Status
   errorMessage?: string
   forceRefresh: () => void
 }
 
+export type PaginatedQueryHookResults<Data extends []> =
+  QueryHookResults<Data> & {
+    /**
+     * Tracks load state for new pages loading in (not the initial page; initial page can be tracked via status).
+     * Only used after calling loadMore() first
```

Comment:
this comment is hard to parse

maybe something like:
> True when additional pages of data are loading. Use instead of status for pages beyond the first, e.g. after calling loadMore().

Reply:
Nice I like that better; yeah I agree I struggled wording this a bit.